python/emitter.js

```diff
@@ -114,6 +114,15 @@
         addHook('deps', 'from applitools.common.ultrafastgrid import DesktopBrowserInfo, IosDeviceInfo, ChromeEmulationInfo, ScreenOrientation')
         addHook('beforeEach', python`    conf.add_browser(${{value: test.config.browsersInfo, type: 'BrowsersInfo'}})`)
     }
+    if ("batch" in test.config) {
+        if ("id" in test.config.batch) {
+            addHook('deps', 'from datetime import datetime')
+            addHook('beforeEach', python`    conf.batch.id = ${test.config.batch.id}`)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
            addHook("beforeEach", python`    conf.batch.id = ${test.config.batch.id}`)
```